content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -455,6 +455,14 @@ The presence of this annotation on a Job indicates that the control plane is
 [tracking the Job status using finalizers](/docs/concepts/workloads/controllers/job/#job-tracking-with-finalizers).
 You should **not** manually add or remove this annotation.
 
+### scheduler.alpha.kubernetes.io/defaultTolerations {#scheduleralphakubernetesio-defaulttolerations}
+
+Example: `scheduler.alpha.kubernetes.io/defaultTolerations: '[{"operator": "Equal", "value": "value1", "effect": "NoSchedule", "key": "dedicated-node"}]'`
+
+Used on: Namespace
+
+This annotation requires the [PodTolerationRestriction](/docs/reference/access-authn-authz/admission-controllers/#podtolerationrestriction) admission controller to be enabled. This annotation key allows assigning tolerations to a namespace and any new pods created in this namespace would get these tolerations added.
```

Comment:
What format is the value of the annotation? For example, is it TOML? (I know it's not, I'm just using that as an example).

Reply:
Appears to be JSON

Reply:
Let's document the format. I agree that it looks like JSON.

Reply:
yes, it's a JSON serialization matching the Pod `spec.tolerations` format (array of Toleration items - https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#toleration-v1-core)

Reply:
https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#scheduling has a reference for this too.